util/pace/util/_legacy_restart.py

```diff
@@ -38,29 +42,43 @@ def open_restart(
     Returns:
         state: model state dictionary
     """
+
     if tracer_properties is None:
         restart_properties = RESTART_PROPERTIES
+        restart_properties.update(extra_restart_properties)  # add microphysics
+        # original restart properties don't have that stuff
     else:
         restart_properties = {**tracer_properties, **RESTART_PROPERTIES}
+        restart_properties.update(extra_restart_properties)
+
```

Comment:
```suggestion
    else:
        restart_properties = {**tracer_properties, **RESTART_PROPERTIES}
    restart_properties.update(extra_restart_properties)  # add microphysics
    # original restart properties don't have that stuff

```

Reply:
I didn't really touch this at all - this was all Oliver :D